radiotools/coreas/generate_coreas_sim.py

```diff
@@ -4,7 +4,9 @@
 import numpy as np
 from radiotools import coordinatesystems
 from radiotools import helper as hp
-from past.builtins import xrange
+from radiotools.atmosphere import models
+import sys
+from miniradiotools.utils.cherenkov_radius import get_cherenkov_radius_model_from_depth
```

Comment:
Is that a new external dependency

Reply:
yeah, I noticed it after the original changes and imported it from another another repository in order to use this branch short-term, but I need to fix it so it isn't using anything external. See my last message about the new functions